xclim/indicators/land/_snow.py

```diff
@@ -32,36 +38,105 @@ class SnowWithIndexing(ResamplingIndicatorWithIndexing):
 
 
 snow_cover_duration = SnowWithIndexing(
-    title="Snow cover duration",
+    title="Snow depth cover duration",
     identifier="snow_cover_duration",
     units="days",
     long_name="Number of days with snow depth at or above threshold",
     description="The {freq} number of days with snow depth greater than or equal to {thresh}.",
     abstract="Number of days when the snow depth is greater than or equal to a given threshold.",
     compute=xci.snow_cover_duration,
+    _version_deprecated="0.41.0",
+)
+
+snow_depth_cover_duration = SnowWithIndexing(
+    title="Snow depth cover duration",
+    identifier="snow_depth_cover_duration",
+    units="days",
+    long_name="Number of days with snow depth at or above threshold",
```

Comment:
```suggestion
    long_name="Snow cover duration",
```
Same comment as in french.